src/main/java/no/sikt/nva/pubchannels/dataporten/Journal.java

```diff
@@ -0,0 +1,78 @@
+package no.sikt.nva.pubchannels.dataporten;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import java.net.URI;
+import no.sikt.nva.pubchannels.Immutable;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+/* default */ final class Journal implements Immutable {
```

Comment:
Can we call this DataportenJournal that implements an interface called ThirdPartyJournal?